### PR TITLE
HA updates

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

File: content/k3s/latest/en/installation/_index.md [hunk 1]
```diff
@@ -5,20 +5,15 @@ weight: 20
 
 This section contains instructions for installing K3s in various environments. Please ensure you have met the [Node Requirements]({{< baseurl >}}/k3s/latest/en/installation/node-requirements/) before you begin installing K3s.
 
-### Installation Options
+[Installation and Configuration Options]({{< baseurl >}}/k3s/latest/en/installation/install-options/) provides guidance on the options available to you when installing K3s.
 
-*   [Single Server Installation]({{< baseurl >}}/k3s/latest/en/installation/single-server/)
 
-	Install K3s on a single Linux host. Single server installs are recommended for development, test, or production environments where the cluster doesn't have to be readily available for a user-base and some downtime is acceptible.
+[High Availability with an External DB]({{< baseurl >}}/k3s/latest/en/installation/ha/) details how to setup an HA K3s cluster backed by an external datastore such as MySQL or PostgreSQL.
```
Comment (davidnuzik) [inline content/k3s/latest/en/installation/_index.md]: Nitpick: We support etcd as well, should we include this here?
e.g.
> [High Availability with an External DB]({{< baseurl >}}/k3s/latest/en/installation/ha/) details how to setup an HA K3s cluster backed by an external datastore such as MySQL, PostgreSQL, or etcd.

File: content/k3s/latest/en/installation/_index.md [hunk 2]
```diff
@@ -5,20 +5,15 @@ weight: 20
...
 
-*   [High Availability (HA) Installation]({{< baseurl >}}/k3s/latest/en/installation/ha/)
+[High Availability with Embedded DB (Experimental)]({{< baseurl >}}/k3s/latest/en/installation/ha/) details how to setup an HA K3s cluster that leverages a built-in distributed database.
```
Comment (davidnuzik) [inline content/k3s/latest/en/installation/_index.md]: Important: Link needs to be fixed.
e.g.
> [High Availability with Embedded DB (Experimental)]({{< baseurl >}}/k3s/latest/en/installation/ha-embedded/)

File: content/k3s/latest/en/installation/ha-embedded/_index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: "High Availability with Embedded DB (Experimental)"
+weight: 40
+---
+
+K3s is now previewing support for running a highly available control plane without the need for an external database. This means there is no need to manage an external etcd or SQL datastore in order to run a reliable production-grade setup. While this feature is currently experimental, we expect it to be the primary architecture for running HA K3s clusters in the future.
+
+This architecture is achieved by embedding a dqlite database within the K3s server process. DQLite is short for "distributed SQLite." According to https://dqlite.io, it is *"a fast, embedded, persistent SQL database with Raft consensus that is perfect for fault-tolerant IoT and Edge devices."*  This makes it a natural fit for K3s.
```
Comment (davidnuzik) [inline content/k3s/latest/en/installation/ha-embedded/_index.md]: Minor: Remove the asterisks or modify the last sentence on this line if needed. Hugo renders only as italics currently and there is no space after the last double quote when I test locally.

File: content/k3s/latest/en/installation/datastore/_index.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+And the following example could be used to connect to a MySQL database using client certificate authentication:
+```
+K3S_DATASTORE_ENDPOINT='mysql://username:password@tcp(hostname:3306)/k3s-db' \
+K3S_DATASTORE_CERTFILE='/path/to/client.crt' \
+K3S_DATASTORE_KEYFILE='/path/to/client.key' \
+k3s server
+```
+
+### Embedded DQLite for HA (preview)
```
Comment (davidnuzik) [inline content/k3s/latest/en/installation/datastore/_index.md]: Nitpick: In other areas we call it "experimental" not "preview"

File: content/k3s/latest/en/installation/airgap/_index.md [hunk 5]
```diff
@@ -75,4 +75,3 @@ Upgrading an air-gap environment can be accomplished in the following manner:
 1. Download the new air-gap images (tar file) from the [releases](https://github.com/rancher/k3s/releases) page for the version of K3s you will be upgrading to. Place the tar in the `/var/lib/rancher/k3s/agent/images/` directory on each node. Delete the old tar file.
 2. Copy and replace the old K3s binary in `/usr/local/bin` on each node. Copy over the install script at https://get.k3s.io (as it is possible it has changed since the last release). Run the script again just as you had done in the past with the same enviornment variables.
```
Comment (davidnuzik) [inline content/k3s/latest/en/installation/airgap/_index.md]: Minor: Could you fix this pre-existing spelling error in 2nd to last word "enviornment" > "environment".

**Comment by cjellick** [comment]
need to update diagram
change compliant to conformant

**Comment by davidnuzik** [comment]
LGTM

**Comment by cjellick** [comment]
https://github.com/rancher/k3s/issues/1082
https://github.com/rancher/k3s/issues/1059

**Pull request merged**